src/html/ReqMgr/templates/doc.tmpl

```diff
@@ -86,8 +103,17 @@ function Activate(tag, hidetags) {
     }
     ShowTag('edit-'+tag);
 }
+function UpdateTaskParams() {
+    var parameters = \$('#taskParams').val();
+    ajaxRequest('$base/data/splitting/$rid', parameters, 'POST');
```

Comment:
Seangchan,
I'm not really sure how to json'ify input form parameters, but googling I found
that javascript supports JSON.Parse. So I applied the change and updated PR.
Please check how it works.
Valentin.

On  0, ticoann <notifications@github.com> wrote:
> > @@ -86,8 +103,17 @@ function Activate(tag, hidetags) {
> >      }
> >      ShowTag('edit-'+tag);
> >  }
> > +function UpdateTaskParams() {
> > +    var parameters = \$('#taskParams').val();
> > +    ajaxRequest('$base/data/splitting/$rid', parameters, 'POST');
> 
> parameters doesn't seems to be json encoded
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6605/files#r53517786